department-of-veterans-affairs/caseflow
# Fix contested typo in Issue Category Dropdown

Resolves #15180

### Description
Fix typo for the word 'contested'

### Acceptance Criteria
- [ ] The word 'Contested' is spelled correctly in the Issues Category dropdown  

### Testing Plan
1. Select 'Decision Review Request' as the Intake form 
2. Follow through earch and Review Steps
3. on Step 4, click 'Add Issue'
4. Select 'Compensation' as the Benefit Type
5. Select the 'Issue Category' dropdown
6. Verify that 'compensated' is spelled correctly

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1349" alt="Screen Shot 2020-09-04 at 3 01 26 PM" src="https://user-images.githubusercontent.com/68879427/92289518-9d1b4700-eec5-11ea-8797-6d192044c03b.png">|<img width="1284" alt="Screen Shot 2020-09-04 at 3 03 40 PM" src="https://user-images.githubusercontent.com/68879427/92289529-a2789180-eec5-11ea-96df-14e58b9c68ff.png">
